File: ios/BullsEye/GuideStream/GuideStreamViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(GuideStreamViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(rtspType, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(onPlaying, RCTDirectEventBlock)
RCT_EXPORT_VIEW_PROPERTY(onRecordComplete, RCTDirectEventBlock)
@end

File: ios/BullsEye/GuideStream/GuideStreamViewManager.swift
import React
import UIKit

@objc(GuideStreamViewManager)
final class GuideStreamViewManager: RCTViewManager {
    override func view() -> UIView! {
        return GuideStreamView()
    }

    override class func requiresMainQueueSetup() -> Bool {
        return true
    }
}
